tests/api/v1/endpoints/test_privacy_request_endpoints.py

```diff
@@ -715,6 +720,41 @@ def test_verbose_privacy_request_embed_limit(
             ExecutionLog.privacy_request_id == privacy_request.id
         ).delete()
 
+    def test_get_privacy_requests_csv_format(
+        self, db, generate_auth_header, api_client, url, privacy_request, user
+    ):
+        reviewed_at = datetime.now()
+        created_at = datetime.now()
+
+        privacy_request.created_at = created_at
+        privacy_request.status = PrivacyRequestStatus.approved
+        privacy_request.reviewed_by = user.id
+        privacy_request.reviewed_at = reviewed_at
+        privacy_request.cache_identity({"email": "email@example.com"})
```

Comment:
It would be good to cover the scenario where we have a dict of multiple identities in case the commas there conflict with the delimiter somehow.

Reply:
great testing addition 👍